.github/workflows/spelling.yml

```diff
@@ -0,0 +1,22 @@
+name: Spellcheck
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: cSpell
+        run: |
+          echo "::add-matcher::.github/workflows/cspell-problem-matcher.json"
```

Comment:
curious about how does this works? is that a feature provided by cspell or GitHub?

Reply:
It's a feature of GitHub. There is a little description of them here https://github.com/actions/toolkit/blob/master/docs/problem-matchers.md